test/scala/com/github/johnynek/bazel_deps/ParseGeneratedDocTest.scala

```diff
@@ -20,7 +20,7 @@ class ParseGeneratedDocTest extends FunSuite {
 
     forAll(genList) { lp =>
       val output = Dependencies.normalize(lp)
-      assert(lp.size <= output.size)
+      assert(lp.size >= output.size)
```

Comment:
Is it bad that this didn't fail in tests if the law was incorrect? (build looks green, but obviously scalacheck style so might not mean much?)

Reply:
yeah, I think the scalacheck is *very* rarely hitting cases where minimization are running, so basically only the handwritten cases are exercising.

It would be nice to make a scala-check `Gen` that makes minimization more likely.